resources/views/stores/show.blade.php

```diff
@@ -96,68 +99,64 @@
-                    @if($store->recommend_image !== null)
                     <img src="{{$store->recommend_image}}" alt="おすすめ商品">
-                    @else
-                    <img class="post-img" src="/css/noimage.png" alt="NO_IMAGE">
-                    @endif
-                    <p>{{$store->recommend_product}}</p>
-                </div>
-            </div>{{-- recommend_shops --}}
+        @endif
+        <p>{{$store->recommend_product}}</p>
+        </div>
+        <div class="detail-link">
+            @if($store->recommend_image == null)
+            <img class="post-img" src="../css/noimage.png" alt="NO_IMAGE">
+            @else
+            <img src="{{$store->recommend_image}}" alt="おすすめ商品">
+            @endif
+            <p>{{$store->recommend_product}}</p>
+        </div>
+        <div class="spacer"></div>
+        <div class="detail-link">
+            @if($store->recommend_image == null)
+            <img class="post-img" src="../css/noimage.png" alt="NO_IMAGE">
+            @else
+            <img src="{{$store->recommend_image}}" alt="おすすめ商品">
+            @endif
+            <p>{{$store->recommend_product}}</p>
+        </div>
+        <div class="detail-link">
+            @if($store->recommend_image == null)
+            <img class="post-img" src="../css/noimage.png" alt="NO_IMAGE">
+            @else
+            <img src="{{$store->recommend_image}}" alt="おすすめ商品">
+            @endif
+            <p>{{$store->recommend_product}}</p>
+        </div>
+        </div>recommend_shops --}}
         </div>
         {{-- 口コミ一覧 --}}
         <div class="review">
             <div class="posting">
                 <h3>口コミ一覧</h3>
-                <form action="{{ route('reviews.create') }}" method="GET">
-                    @csrf
-                    <input type="hi
-                    dden" name="" id="">
-                    <button type="submit" class="btn-text-danger">
-                        <img src="./css/rogo_addreview.jpg" alt="口コミ鉛筆" class="rogo">te
-                    </button>
-                    <div class="word font">
-                        <p>口コミを投稿する</p>
-                    </div>
+                <button type="submit" class="btn-text-danger">
+                    <img src="{{ asset('css/rogo_addreview.jpg') }}" alt="口コミ鉛筆" class="rogo">
+                </button>
+                <div class="word font">
+                    <a href="/reviews/create?store_id={{$store->id}}"></a>
+                    <p>口コミを投稿する</p>
+                </div>
             </div>
             <div class="review-wrapper">
                 {{-- 投稿した内容を表示 --}}
-                @foreach ($store->$reviews as $review)
+                @foreach ($store->reviews as $review)
                 <div class="review-details">
                     <dl class="posted-data">
-                        <dt class="post-title row">{{$review->name}}さんの口コミ</dt>
+                        <dt class="post-title row">{{$review->user->name}}さんの口コミ</dt>
```

Comment:
退会した？された？、、日本語怪しいですが...

```suggestion
@if($review->user)
    <dt class="post-title row">{{$review->user->name}}さんの口コミ</dt>
@else
    <dt class="post-title row">退会されたユーザーの口コミ</dt>
@endif
```